### PR TITLE
163 adding risk adjustment example

### PR DESCRIPTION
**Pull request by chase-jones**
@sarah-tuva pinging for awareness. This is first pass at providing single patient example for this issue. Open to feedback or changes you would like.

@aneiderhiser Let me know if this has the types of graphics / charts you were envisioning.

### CODE REVIEW

File: docs/risk-adjustment/cms-hccs.md [hunk 1]
```diff
@@ -118,6 +118,136 @@ of the steps:
...
+
+#### But wait! There's more
+
+In the above example, it only looked the scores and weights for a single risk adjustment model, v28.
+However, for medicare advantage organizations in the year 2024, final funded risk is not based solely
+on the outputs of the v28 risk adjustment model. There is a transition period where risk will be 
+determined with a blended model, where 33% of the risk score will be weighted with the v28 model
+and 67% of the risk score will be weighted with the v24 risk adjustment model. 
+
+What does this mean? This means we have to go back and repeat the steps prior to "Brining it all together"
```
Comment (sarah-tuva) [inline docs/risk-adjustment/cms-hccs.md:237]:
```suggestion
What does this mean? This means we have to go back and repeat the steps prior to "Bringing it all together"
```

**Comment by aneiderhiser** [comment]
@chase-jones this is AWESOME!  Fantastic addition.  I'm going to merge this.  Then I have a couple thoughts on re-organizing a few things that I'll open a separate PR for.

**Review by sarah-tuva — CHANGES_REQUESTED** [review]
@chase-jones Incredible job! 👏  Thank you so much for your work in putting this example together with screenshots. I can imagine how much time this took. I just saw a few typos at the end. I even learned a few things. Thanks again! 🙌

**Comment by chase-jones** [comment]
Glad to hear it! There's still some outstanding detail here that is missing, but not sure how that is a priority for you all or if I should make it an issue.

Some examples (off the top of my head and in no particular order):

* Date ranges and dates of service 
* MMR initial enrollment and 26 sweep 
* How age is exactly determined
* Rounding / sig figs
* Nitty gritty in regards to eligibility

**Comment by aneiderhiser** [comment]
@chase-jones those add'l details all make sense to me.  Feel free to create an issue to keep track.  

I'll let @sarah-tuva re-review your latest commit and then we can merge this in.

**Review by sarah-tuva — APPROVED** [review]
Looks great to me! I can merge it in if we're ready to go. It sounds like we'll create an issue to track the additional examples we want to add. Thanks again @chase-jones! 🌟

**Pull request merged**